pom.xml

```diff
@@ -197,18 +197,41 @@
                </environments>
             </configuration>
     </plugin>
-	<plugin>
-		<groupId>org.eclipse.tycho</groupId>
-		<artifactId>tycho-surefire-plugin</artifactId>
-		<version>${tycho.version}</version>
-		<configuration>
-			<useUIHarness>true</useUIHarness>
-			<useUIThread>false</useUIThread>
-			<product>org.eclipse.platform.ide</product>
-		</configuration>
-	</plugin>
-
-      </plugins>
+    <plugin>
+	    <groupId>org.eclipse.tycho</groupId>
+	    <artifactId>tycho-surefire-plugin</artifactId>
+	    <version>${tycho.version}</version>
+	    <configuration>
+		    <useUIHarness>true</useUIHarness>
+		    <useUIThread>false</useUIThread>
+		    <product>org.eclipse.platform.ide</product>
+		    <skipTests>true</skipTests>
```

Comment:
@wolfgang-ch By default, I have set it so that the unit tests do not run because it takes more time for the build to complete. This way, when you run the build on your machine or for the new MT releases, it takes the same time as before. It is only activated for the builds running on GitHub. But you know how to activate it if you want the unit tests to run